inference/scripts/build-wasm.py

```diff
@@ -158,8 +163,9 @@ def run_shell(command):
         print("\n🏃 Running CMake for Bergamot\n")
         run_shell(f"emcmake cmake -DCOMPILE_WASM=on -DWORMHOLE=off {flags} {INFERENCE_PATH}")
 
-        print("\n🏃 Building Bergamot with emmake\n")
-        run_shell(f"emmake make -j {multiprocessing.cpu_count()}")
+        cores = args.j if args.j else multiprocessing.cpu_count()
+        print(f"\n🏃 Building Bergamot with emmake using {cores} cores\n")
+        run_shell(f"emmake make -j {cores}")
```

Comment:
This work-around seems fine. Suggestion: Can you add a try/except block to the fallible build step so that when it fails, it suggests trying this.